### ISSUE TITLE
QA images for alt text descriptions

### ISSUE THREAD

**Issue by jaimemears** [jywarren/seeing-lost-enclaves#27]
Reference: https://staff.loc.gov/sites/digital-accessibility/creating-content/image-description-alt-text/

Image description is a short text description of the information communicated in an image. It is most often provided as something called “alternative text” or “alt text”. This description makes that information available in the image to people who are blind, have a metered internet connection, and whose email security settings prevent images from loading. In other words, alt text is important! Adding it is one of the most important things you can do to make your content accessible and inclusive. Images without alt text are one of the most frequent barriers encountered by people who are blind or low-vision.

If you are using an image, you are responsible for including alt text. Alt text isn’t usually visible on the page. It is included in the page mark-up and screen readers will read it aloud. (Screen readers are assistive technology that read out the structure and text of software, web pages, and apps.) When an image does not have alt text, a screen reader user will know that there is an image present, but that they are not able to access information because the author didn’t take the time to make things accessible.

Alt text may appear visibly on the page if an image isn’t loaded because of a poor internet connection or security settings. This helps communicate your meaning to everyone. Search engines also use alt text to improve search results.

The Library provides guidance for image description and alt text for a variety of social media platforms. You can find this information and more on the [Social Media: Key Documents page](https://staff.loc.gov/sites/webproduction/social-media-key-documents/).
